mmselfsup/models/memories/interclr_memory.py

```diff
@@ -0,0 +1,254 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import numpy as np
+import torch
+import torch.distributed as dist
+import torch.nn as nn
+from mmcv.runner import BaseModule, get_dist_info
+from sklearn.cluster import KMeans
+
+from mmselfsup.utils import AliasMethod
+from ..builder import MEMORIES
+
+
+@MEMORIES.register_module()
+class InterCLRMemory(BaseModule):
+    """Memory bank for InterCLR.
+
+    Args:
+        length (int): Number of features stored in the memory bank.
+        feat_dim (int): Dimension of stored features.
+        momentum (float): Momentum coefficient for updating features.
+        num_classes (int): Number of clusters.
+        min_cluster (int): Minimal cluster size.
+    """
+
+    def __init__(self, length, feat_dim, momentum, num_classes, min_cluster,
+                 **kwargs):
+        super(InterCLRMemory, self).__init__()
```

Comment:
```suggestion
        super().__init__()
```

Reply:
This follows the same style in `ODCMemory` and `SimpleMemory`